.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    branches:
+      - stage
+
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: pull-request
+        uses: actions/checkout@v3
+      - name: Create Pull Request
+        uses: diillson/auto-pull-request@v1.0.1
```

Comment:
Can we clone this action? Seems like the safer thing to do & we will likely need to adapt it anyway